javascript/javascript1/week2/students.js

```diff
@@ -0,0 +1,50 @@
+const class07Students = ['Anna','Evan','Nobin','Anju'];
+function addStudentToClass(studentName) {
```

Comment:
Looking at your solution to this, I think it works better than the one I shared with you. Good job on this!